tests/invalid/MethodCall_Invalid_2.whiley:7: unable to resolve name (no match for f(&int,int))
	found: MethodCall_Invalid_2:f : method(dummy,int)->(int)
    f(_this, 1)
    ^
